src/cli.mjs

```diff
@@ -94,7 +94,12 @@ function buildYargs(argvInput) {
         alias: "d",
         type: "string",
         description: "a database path, including region",
-        // required: true,
+      },
+      role: {
+        alias: "r",
+        type: "string",
+        description: "a role",
+        default: "admin",
```

Comment:
does this apply for every CLI command, or should it be shared to a smaller set of commands?

Reply:
everything that will be using the middleware, i.e. everything that needs a db key.